packages/devextreme-scss/scss/widgets/material/chat/_sizes.scss

```diff
@@ -72,10 +92,16 @@ $chat-messagelist-day-header-first-padding-top: 20px !default;
   $chat-errors-row-gap: 2px !default;
   $chat-error-padding-block: 5px !default;
 
-  $chat-messagelist-empty-icon-box-size: 48px !default;;
+  $chat-messagelist-empty-icon-box-size: 48px !default;
   $chat-messagelist-empty-icon-size: 32px !default;
   $chat-messagelist-empty-icon-margin-bottom: 14px !default;
   $chat-messagelist-empty-row-gap: 2px !default;
   $chat-messagelist-empty-message-font-size: 14px !default;
   $chat-messagelist-empty-prompt-font-size: 12px !default;
+
+  $chat-typingindicator-circle-size: math.div(8, 13) * 1em !default;
```

Comment:
Instead of number value we should use variables, in this case it should be $material-base-font-size

Reply:
Let's consider it in future task. Kindly create task for this

Reply:
We don't have those variables. 

If you mean the variable for font-size, it cannot be used because it contains a value in pixels:

https://github.com/DevExpress/DevExtreme/blob/24_2/packages/devextreme-scss/scss/widgets/fluent/_sizes.scss#L37

When interpreting scss styles, an error will occur because you are trying to mix px and em units.

Also, note that we only care about the ratio of 8 to 13 (or other), we only care about the ratio because the circle sizes will still end up being calculated relative to the font size inherited by this elements. So there is no error or oversight here, because all that matters to you at this stage is the magnification ratio.